EMOlater/backend/models.py

```diff
@@ -1,13 +1,12 @@
 from django.db import models
 
 class SurveyRepsponse(models.Model):
-    body = models.CharField(max_length=50)
     user_id = models.IntegerField()
     timestamp = models.DateField()
-    family = models.BooleanField()
-    partner = models.BooleanField()
-    friends = models.BooleanField()
-    colleagues = models.BooleanField()
+    family = models.BooleanField(default=False)
```

Comment:
Is this field ever optional?  it should only have a default value if it's sometimes acceptable for a user to not specify it